### PR TITLE
test(link, loader, inlineinputs): add missing cypress tests (FE-2351)

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Add missing cypress test for loader default component,
Replaced Given-When-Then steps in Link component,
Add new background name for inlineInputs

### Checklist

<del> - [ ] Release notes (Conventional Commits)  <del>
<del>  - [ ] Unit tests <del>  
- [x] Cypress automation tests

<del>  - [ ] Storybook added or updated <del>  
<del>  - [ ] Theme support <del>  
<del>  - [ ] Typescript `d.ts` file added or updated <del>  

### Testing instructions

run: npm run test-cypress and choose 
loader.feature,
inlineInputs.feature,
link.feature

### CODE REVIEW

File: cypress/features/regression/loader.feature [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
+      | small | 17     |
+      | large | 19     |
 
   @positive
-  Scenario Outline: I set Loader component size to <size>
-    When I select size to "<size>"
-    Then Loader width and height is set to "<px>"
+  Scenario Outline: Verify size of button with loader
+    Given I select size to "<size>"
+    When I check isInsideButton checkbox
```
Comment (DlgSHi) [inline cypress/features/regression/loader.feature]: I check isInsideButton checkbox - should be a `Given`

File: cypress/features/regression/loader.feature [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
+    Then Loader isInsideButton and backgroundColor is "rgb(0, 128, 93)"
+
+  @positive
+  Scenario: Disabled loader button
+    Given I check isInsideButton checkbox
+    When I uncheck isActive checkbox
+    Then Loader button is disabled
+
+
```
Comment (DlgSHi) [inline cypress/features/regression/loader.feature]: please remove an unnecessary empty line

File: cypress/features/regression/loader.feature [hunk 3]
```diff
@@ -1,36 +1,47 @@
...
 
   @positive
-  Scenario Outline: I set Loader component as to <as>
-    When I select as to "<as>"
-    Then Loader "border-top-color" color is set to "<border-top-color>"
-      And Loader "border-left-color" color is set to "0.7" gradient of "<border-top-color>" base color
-      And Loader "border-right-color" color is set to "0.3" gradient of "<border-top-color>" base color
+  Scenario Outline: I set Loader component size to <size>
+    When I select size to "<size>"
+    Then Loader width is set to "1059px" and height is set to "<height>"
```
Comment (DlgSHi) [inline cypress/features/regression/loader.feature]: could we pass only an integer as a parameter without `px`? We could move `px` inside step

File: cypress/features/regression/loader.feature [hunk 4]
```diff
@@ -1,36 +1,47 @@
...
+      | large | 19     |
 
   @positive
-  Scenario Outline: I set Loader component size to <size>
-    When I select size to "<size>"
-    Then Loader width and height is set to "<px>"
+  Scenario Outline: Verify size of button with loader
+    Given I select size to "<size>"
+    When I check isInsideButton checkbox
+    Then button with loader width is set to "<width>" and height is set to "40px"
```
Comment (DlgSHi) [inline cypress/features/regression/loader.feature]: could we pass only an integer as a parameter without `px`? We could move `px` inside step

File: cypress/support/step-definitions/loader-steps.js [hunk 5]
```diff
@@ -19,3 +30,24 @@ Then('Loader {string} color is set to {string} gradient of {string} base color',
...
   spinner().should('have.css', `${parameter}`, `${color}`);
 });
+
+Then('Loader isInsideButton and backgroundColor is {string}', (color) => {
+  loaderInsideButton().should('be.visible')
+    .and('have.css', 'background-color', color);
+});
+
+Then('Loader button is disabled', () => {
+  loaderInsideButton().should('have.attr', 'disabled');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/loader-steps.js]:
```suggestion
  loaderInsideButton().should('be.disabled').and('have.attr', 'disabled');
```

File: cypress/locators/loader/locators.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// component preview locators
+export const LOADER = '[data-component="loader"]';
+export const LOADER_INSIDE_BUTTON = '[data-component="button"]';
```
Comment (DlgSHi) [inline cypress/locators/loader/locators.js]: I think we already have defined such locator in common or in button `locators.js` file

File: cypress/features/regression/loader.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/loader.feature]: fixed

File: cypress/features/regression/loader.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/loader.feature]: fixed

File: cypress/features/regression/loader.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/loader.feature]: fixed

File: cypress/features/regression/loader.feature (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/features/regression/loader.feature]: fixed

File: cypress/support/step-definitions/loader-steps.js (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/loader-steps.js]: fixed

File: cypress/locators/loader/locators.js (hunk 6 above)
Comment (KatarzynaQA) [reply cypress/locators/loader/locators.js]: fixed

**Pull request merged**